readme.md

```
@@ -1,134 +1,194 @@
+# Proyecto Xambi - Documentación Final
 ## Índice
-
 0. [Ficha del proyecto](#0-ficha-del-proyecto)
 1. [Descripción general del producto](#1-descripción-general-del-producto)
 2. [Arquitectura del sistema](#2-arquitectura-del-sistema)
-3. [Modelo de datos](#3-modelo-de-datos)
+3. [Modelo de datos](#3-modelo-datos)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)